test/verify/check-shell-host-switching

```diff
@@ -123,6 +129,22 @@ class TestHostSwitching(MachineCase, HostSwitcherHelpers):
         self.allow_restart_journal_messages()
         self.allow_hostkey_messages()
 
+    def testSSH(self):
+        b = self.browser
+        m1 = self.machines["machine1"]
+
+        self.login_and_go(superuser=False)
+
+        b.click("#hosts-sel button")
+        self.wait_host_addresses(b, ["localhost"])
+
+        self.add_machine(b, "ssh://10.111.113.2", False, user="admin")
+        self.machine_remove(b, "10.111.113.2", m1, True)
```

Comment:
This only runs the "Add" dialog, but does not ensure it actually worked. You need to do the `self.wait_host_addresses()` and `self.connect_and_wait()` steps to assert that the connection was successful.